### PR TITLE
Add missing content to the release notes

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

The 2.4 release notes lack information about PodPreset deprecation.

Changes proposed in this pull request:

- Add info about PodPreset deprecation

### CODE REVIEW

File: content/blog-posts/2022-06-30-release-notes-2.4/index.md [hunk 1]
```diff
@@ -20,72 +20,160 @@ We’ve also finalized our spring cleaning and completely removed Service Catalo
...
-## Service Management 
+Once the PodPresets are removed, the Secrets consumed by your existing Pods will only last until the restart. Upon the next Pods creation, the auto-injecting mechanism will be gone and you will have to mount the Secrets using these solutions:
+- Mounting Secrets to Kyma Functions
+- Mounting Secrets to Kubernetes Deployments (using either volume or environment variable mounting)
 
+#### Mounting Secrets to Kyma Functions
 
-### Service Catalog removed 
+Mounting Secret data to Kyma Function ensures that the data is preserved in the Function despite the configuration changes you may provide in the future. In this method, all Secret keys become environment variables. You can provide them to your Function using Kyma Dashboard:
+1. Go to the **Functions** view in Kyma Dashboard and select a Function.
```
Comment (NHingerl) [inline content/blog-posts/2022-06-30-release-notes-2.4/index.md:92]:
```suggestion
1. In Kyma Dashboard, go to the **Functions** view and select a Function.
```

File: content/blog-posts/2022-06-30-release-notes-2.4/index.md [hunk 2]
```diff
@@ -20,72 +20,160 @@ We’ve also finalized our spring cleaning and completely removed Service Catalo
...
+
+If you already switched to another solution and want to remove the obsolete CRDs from your cluster, run Service Catalog cleanup script provided in the [Migration Guide](https://kyma-project.io/docs/kyma/2.4/migration-guide-2.3-2.4#service-catalog-cleanup-script) **after** you upgrade from Kyma 2.3 to 2.4.
+
+Service Catalog removal also affects Application Connectivity. We removed some components that interacted with Service Catalog, which means that Application Operator and Application Broker are no longer installed on new clusters. This also means that on fresh clusters, or after executing the migration script we provide, the old Application flow will no longer be present, and you will need to use the new way of connecting the external Systems.
+
+### PodPreset deprecation note
+
+Due to the Service Catalog removal, Kyma will stop manage PodPresets by the end of October, 2022. Make sure to migrate your resources until then.
```
Comment (NHingerl) [inline content/blog-posts/2022-06-30-release-notes-2.4/index.md:83]:
```suggestion
Due to the Service Catalog removal, Kyma will stop managing PodPresets by the end of October, 2022. Make sure to migrate your resources until then.
```

File: content/blog-posts/2022-06-30-release-notes-2.4/index.md [hunk 3]
```diff
@@ -20,72 +20,160 @@ We’ve also finalized our spring cleaning and completely removed Service Catalo
...
+        name: test-secret
+  - name: Name
+    valueFrom:
+      secretKeyRef:
+        key: Name
+        name: test-secret
+```
 
-- Helm Broker 
+Alternatively, you can add the env property by editing Function's specification directly.
```
Comment (NHingerl) [inline content/blog-posts/2022-06-30-release-notes-2.4/index.md:112]:
```suggestion
Alternatively, you can add the env property by editing a Function's specification directly.
```
I think we need an article here

File: content/blog-posts/2022-06-30-release-notes-2.4/index.md [hunk 4]
```diff
@@ -20,72 +20,160 @@ We’ve also finalized our spring cleaning and completely removed Service Catalo
...
+- You can also mount Secrets to Kubernetes Deployments using environment variables:
 
-## Service Mesh 
+  - In the Deployment's **spec.template.spec.containers[]** property, add **envFrom** to every container in which you want to consume a Secret:
+    ```yaml
+    envFrom:
+      - secretRef:
+        name: test-secret
+    ```
+  - If you want to configure environment variables, add env to the **spec.template.spec.containers[]** property to every container in which you want to consume a Secret. See the example:
```
Comment (NHingerl) [inline content/blog-posts/2022-06-30-release-notes-2.4/index.md:145]: Is this an alternative to the instruction in line 139, or a second step afterwards? 
If it's a second step, I'd consider a numbered list.

Comment (klaudiagrz) [reply content/blog-posts/2022-06-30-release-notes-2.4/index.md:145]: It's an alternative.

File: content/blog-posts/2022-06-30-release-notes-2.4/index.md [hunk 5]
```diff
@@ -20,72 +20,160 @@ We’ve also finalized our spring cleaning and completely removed Service Catalo
...
+        name: test-secret
+  - name: Name
+    valueFrom:
+      secretKeyRef:
+        key: Name
+        name: test-secret
+```
 
-- Helm Broker 
+Alternatively, you can add the env property by editing a Function's specification directly.
```
Comment (klaudiagrz) [inline content/blog-posts/2022-06-30-release-notes-2.4/index.md:112]:
```suggestion
Alternatively, you can add the env property by editing the Function's specification directly.
```

File: content/blog-posts/2022-06-30-release-notes-2.4/index.md (hunk 4 above)
Comment (NHingerl) [reply content/blog-posts/2022-06-30-release-notes-2.4/index.md:145]: then all good :D

**Review by NHingerl — APPROVED** [review]

**Pull request merged**